go.mod

```diff
@@ -8,18 +8,18 @@ require (
 	github.com/kubernetes-csi/csi-lib-utils v0.7.0
 	github.com/kubernetes-csi/csi-test v2.0.0+incompatible
 	github.com/kubernetes-csi/external-snapshotter v1.2.1-0.20191220180133-bba358438aee
-	github.com/miekg/dns v1.1.8 // indirect
 	github.com/spf13/pflag v1.0.5
 	google.golang.org/grpc v1.26.0
-	k8s.io/api v0.17.0
-	k8s.io/apimachinery v0.17.1-beta.0
+	k8s.io/api v0.17.3
+	k8s.io/apimachinery v0.17.3
 	k8s.io/apiserver v0.17.0
 	k8s.io/client-go v0.17.0
 	k8s.io/component-base v0.17.0
 	k8s.io/csi-translation-lib v0.17.0
 	k8s.io/klog v1.0.0
 	k8s.io/kubernetes v1.14.0
-	sigs.k8s.io/sig-storage-lib-external-provisioner v4.1.0+incompatible
+	sigs.k8s.io/sig-storage-lib-external-provisioner v4.1.0+incompatible // indirect
```

Comment:
This is kind of strange, we have dependencies on both versions?

Reply:
I had noticed that, too, but because we never actually pull in any code from it (= it's not in `vendor`) ignored it.

This might be a bug in `sig-storage-lib-external-provisioner`:
```
$ go mod why -m sigs.k8s.io/sig-storage-lib-external-provisioner
# sigs.k8s.io/sig-storage-lib-external-provisioner
github.com/kubernetes-csi/external-provisioner/cmd/csi-provisioner
sigs.k8s.io/sig-storage-lib-external-provisioner/v5/controller
sigs.k8s.io/sig-storage-lib-external-provisioner/v5/controller.test
sigs.k8s.io/sig-storage-lib-external-provisioner/controller/metrics
```

=> controller.test (which we don't use) pulls in the unversioned `sigs.k8s.io/sig-storage-lib-external-provisioner` here: https://github.com/kubernetes-sigs/sig-storage-lib-external-provisioner/blob/7d9e8b4c678a803070309639110a4494a4010efe/controller/controller_test.go#L26

Not sure how that works; perhaps `go test` simply maps that to the checked out module source code. Will fix it.

Reply:
Fix is in https://github.com/kubernetes-sigs/sig-storage-lib-external-provisioner/pull/71

Reply:
Thanks! Since we're not actually using it here, we don't need to wait for.